Source/problems/_default_prob_params

```diff
@@ -1 +1,25 @@
 center                          real                  0.0e0_rt                 n                      3
+
```

Comment:
why do we want these defined for all problems?  isn't there a better way to separate it out?

Reply:
Yes, I suppose the more natural thing would be that they only get added in if you use the model parser module, but we don't really have the infrastructure for that do we? I thought there could only be one _prob_params file, and this _default_prob_params file is something I hacked in that allows us to do things like this. If _prob_params was cumulative like the extern _params is, this would work.

Reply:
ah indeed.  I think we can make it cumulative.  I'd prefer that.

Reply:
Due to #1500, these parameters could be moved to the model parser directory, so I have now done so.